File: threedollar-api-core/src/main/java/com/depromeet/threedollar/api/core/ThreeDollarApiCoreRoot.java
package com.depromeet.threedollar.api.core;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

import com.depromeet.threedollar.domain.rds.ThreeDollarDomainRdsRoot;

@ComponentScan(basePackageClasses = {
    ThreeDollarApiCoreRoot.class,
    ThreeDollarDomainRdsRoot.class,
})
@Configuration
public class ThreeDollarApiCoreRoot {

}

File: threedollar-api-core/src/main/java/com/depromeet/threedollar/api/core/ThreeDollarApiCoreRoot.kt
package com.depromeet.threedollar.api.core

import com.depromeet.threedollar.domain.rds.ThreeDollarDomainRdsRoot
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

@ComponentScan(basePackageClasses = [
    ThreeDollarApiCoreRoot::class,
    ThreeDollarDomainRdsRoot::class
])
@Configuration
class ThreeDollarApiCoreRoot
